compiler-plugin-tests/src/test/resources/ballerina_sources/sample_22/service.bal

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2022 WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/websubhub;
+import ballerina/http;
+import ballerina/io;
+
+listener http:Listener httpListener = new http:Listener(10012);
+listener websubhub:Listener functionWithArgumentsListener = check new(httpListener, {
+    secureSocket: {
+        key: {
+            path: "tests/resources/ballerinaKeystore.pkcs12",
+            password: "ballerina"
+        }
+    }
+});
+
+service /websubhub on functionWithArgumentsListener {
+    isolated remote function onRegisterTopic(websubhub:TopicRegistration message)
+                                returns websubhub:TopicRegistrationSuccess|websubhub:TopicRegistrationError {
+        if (message.topic == "test") {
```

Comment:
Seems "test","test1" used in multiple places, we could introduce a constants

Reply:
Since these are test resource for compiler-plugin, will address this in another PR.